frontend/app/templates/components/person-cv-export.hbs

```diff
@@ -7,6 +7,15 @@
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
+        <div class="modal-body">
+          Niederlassung (für Footer):
+          <div class="container-fluid">
+            <PowerSelect @selected={{this.selectedLocation}} @options={{this.locations}} @onFocus={{action "handleFocus" }}
+                            @onBlur={{action "handleBlur" }} @onChange={{action "setLocation"}} as |location|>
```

Comment:
instead of `{{action "handleFocus"}}` use `{{this.handleFocus}}`

Reply:
Good call